src/prefect/client/orchestration/_flow_runs/client.py

```diff
@@ -46,6 +46,9 @@ def create_flow_run(
         tags: "Iterable[str] | None" = None,
         parent_task_run_id: "UUID | None" = None,
         state: "State[R] | None" = None,
+        work_pool_name: str | None = None,
```

Comment:
```suggestion
        work_pool_name: None | str = None,
```

I think this convention is preferred especially if you're using Google style annotation here.

Reply:
Thanks for the review @nadzhou! Looking at the Google Python style guide, all the examples follow the `str | None` pattern, even if the default value is `None` ([ref](https://github.com/PrefectHQ/prefect/blob/24ff913a96c33dd2bd685fabd7951d1d0affde90/src/prefect/transactions.py#L454)), so I'm going to leave this as is. I'm super stoked to have you checking out this PR, though!